### ISSUE TITLE
/lib64/libc.so.6: version `GLIBC_2.32' not found 

### ISSUE THREAD

**Issue by scydas** [NVIDIA/kubevirt-gpu-device-plugin#81]
version: 1.2.3

nvidia gpu vfio-pci enabled
![image](https://github.com/NVIDIA/kubevirt-gpu-device-plugin/assets/22952515/4fa3c256-44a8-464e-9179-89096708405e)

kubevirt-gpu-device-plugin log
![image](https://github.com/NVIDIA/kubevirt-gpu-device-plugin/assets/22952515/6b47e30c-981e-455e-8930-a63a7bc343b4)

**Comment by tswangdi** [comment]
The image is outdated:

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: device-plugin
spec:
  containers:
    - name: busybox
      image: nvcr.io/nvidia/kubevirt-gpu-device-plugin:v1.2.3
      resources:
        limits:
          cpu: 1
          memory: 1Gi
      command:
        - sleep
        - "1000000"
```

```bash
% kubectl exec -it device-plugin -- bash
[root@device-plugin /]# ldd --version
ldd (GNU libc) 2.28
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

**Comment by rthallisey** [comment]
Fixed by https://github.com/NVIDIA/kubevirt-gpu-device-plugin/pull/82.  Release https://github.com/NVIDIA/kubevirt-gpu-device-plugin/releases/tag/v1.2.4 has the fix

**Comment by scydas** [comment]
> Fixed by #82. Release https://github.com/NVIDIA/kubevirt-gpu-device-plugin/releases/tag/v1.2.4 has the fix

extremely grateful
